subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinMultiplatformPluginSmokeTest.groovy

```diff
@@ -48,7 +48,7 @@ class KotlinMultiplatformPluginSmokeTest extends AbstractKotlinPluginSmokeTest {
         result.task(':tasks').outcome == SUCCESS
 
         where:
-        kotlinVersion << TestedVersions.kotlin.versions
+        kotlinVersion << TestedVersions.kotlin.versions.dropWhile { !it.startsWith('1.9') }
```

Comment:
❓ This contradicts "Gradle is tested with Kotlin 1.6.10 through 1.9.20-Beta.", right? Is there an alternative for other versions for this test?

Reply:
Good catch. It's a remnant of a local modification. Removed!